README.md

```diff
@@ -1,14 +1,12 @@
-Bitcoin Core integration/staging tree
+**Bitcoin Core Integration/Staging Tree**
```

Comment:
There is no need to bold titles in markdown. The idea is to leave it up to the frontend how a certain style should look.